ast/location.go

```diff
@@ -30,8 +30,9 @@ type Source struct {
 
 // Location represents a single location in an (unspecified) file.
 type Location struct {
-	Line   int
-	Column int // bytes? runes? real columns?
+	Line int
+	// TODO(sbarzowski) Should be runes (unicode codepoints), but it's bytes.
```

Comment:
We agreed that since clang uses bytes, it's OK for us to use bytes too :)

Reply:
FYI this comment has been changed already, this is an outdated version.